CHANGES

```diff
@@ -1,6 +1,47 @@
 1.6 unreleased
 ==============
 
+Notable User Facing Changes
+---------------------------
+
+- CUDA kernels using constant __local blocks are now ABI incompatible with
+  previous release. Users need to delete their pocl cache.
+
+
+Notable Bug Fixes
```

Comment:
Is that really a bug fix? I'd put it under Optimizations or such title. Also can you summarize the performance improvement in terms of percentages instead of listing all the benchmarks. Great work!